packages/tss-lib/tsconfig.build.json

```diff
@@ -1,4 +0,0 @@
-{
```

Comment:
This is probably still necessary, please double check.

Reply:
I think we are fine without this. We have the tsconfig.json.